CHANGELOG.md

```diff
@@ -1,3 +1,39 @@
+### citus v8.1.0 (December 17, 2018) ###
+
+* Turns on ssl by default for new installations of citus
+
+* Restricts SSL Ciphers to TLS1.2 and above
+
+* Adds support for INSERT INTO SELECT..ON CONFLICT/RETURNING via coordinator
+
+* Adds support for round-robin task_assignment_policy for queries
+
+* Adds support for SQL tasks using worker_execute_sql_task UDF with task-tracker
+
+* Adds support for VALIDATE CONSTRAINT queries
+
+* Adds support for disabling hash aggregate with HLL
+
+* Adds user ID suffix to intermediate files generated by task-tracker
+
+* Disallows GROUPING SET clauses in subqueries
+
+* Do not restrict user-defined group ID in node addition functions
+
+* Relaxes multi-shard modify locks when enable_deadlock_prevention is disabled
+
+* Cleans up UDFs and revokes unnecessary permissions
+
+* Improves security in task-tracker protocol
+
+* Improves permission checks in internal DROP TABLE functions
+
+* Improves permission checks in cluster management functions
+
+* Fixes permission checks
```

Comment:
This sounds too generic. Maybe use the title itself? `Fix permissions checks in lesser-known UDFs`

Reply:
The original description was `Fixes permission checks in mark_tables_colocated, upgrade_to_reference_table, lock_relation_if_exists` which is way too long. I am using your idea instead

Reply:
Should I merge these two and say `Cleans up UDFs and fixes permissions` ? What do you think @onderkalaci ?

Reply:
Merging them into one makes sense